asr1k_neutron_l3/models/netconf_yang/l3_interface.py

```diff
@@ -227,6 +230,11 @@ def to_dict(self, context):
             else:
                 vbi[L3Constants.NTP][xml_utils.OPERATION] = NC_OPERATION.REMOVE
 
+        if self.arp_timeout and int(self.arp_timeout) > 0:
+            vbi[L3Constants.ARP] = {L3Constants.TIMEOUT: int(self.arp_timeout)}
+        else:
```

Comment:
Wouldn't this lead to us sending a `delete` everytime we sync an interface, even if arp timeout is not in use and has not been configured either?

Reply:
Yes, we'd always send the delete requests (or better: state description) with the configuration, but as in most cases this is already the current state, confd on the asr1k shouldn't issue a cli command for it, i.e. this shouldn't be a costly operation.